corehq/blobs/s3db.py

```diff
@@ -291,7 +185,7 @@ def maybe_not_found(throw=None):
         yield
     except ClientError as err:
         if not is_not_found(err):
-            datadog_counter('commcare.blobdb.notfound')
             raise
+        datadog_counter('commcare.blobdb.notfound')
```

Comment:
how come this moved here?

Reply:
The previous placement was incorrect. Notice the double-negative `not is_not_found ~> is_found`